### PR TITLE
DIRECTOR: Add detection for many Director games

### PR DESCRIPTION
**Pull request by trembyle**
This incorporates changes from closed #2308.

@sev- You had asked me to create a new macro for non-English languages, as well as perform some cleanup and organization. Commit 7ef05794bd80a5469d39ea51194eb26cf842720b should hopefully address that.

### CODE REVIEW

**Comment by trembyle** [comment]
Retry DeepCode

**Comment by sev-** [comment]
Did you know that you could just push or force push into the branch, and PR will get automagically updated?

File: engines/director/detection_tables.h [hunk 1]
```diff
@@ -31,6 +31,10 @@ namespace Director {
 #define WINGAME(t,e,f,m,s,v) 	GENGAME_(t,e,f,m,s,Common::EN_ANY,Common::kPlatformWindows,ADGF_NO_FLAGS,v)
 #define MACDEMO(t,e,f,m,s,v) 	GENGAME_(t,e,f,m,s,Common::EN_ANY,Common::kPlatformMacintosh,(ADGF_MACRESFORK|ADGF_DEMO),v)
 #define WINDEMO(t,e,f,m,s,v) 	GENGAME_(t,e,f,m,s,Common::EN_ANY,Common::kPlatformWindows,ADGF_DEMO,v)
+#define MACGAME_1(t,e,f,m,s,l,v) GENGAME_(t,e,f,m,s,l,Common::kPlatformMacintosh,ADGF_MACRESFORK,v)
```
Comment (sev-) [inline engines/director/detection_tables.h:34]: ehh, I meant lowercase 'L', not 'one' :D. MACGAME_l (for language)

**Comment by sev-** [comment]
Thanks, merging. I'll change MACGAME_l by myself.

File: engines/director/detection_tables.h (hunk 1 above)
Comment (trembyle) [reply engines/director/detection_tables.h:34]: Thanks! I'll get glasses one of these days ...

**Pull request merged**